<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>364</width>
    <height>345</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="LineEdit" name="account_edit">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>110</y>
     <width>211</width>
     <height>21</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>example@example.com</string>
   </property>
  </widget>
  <widget class="LineEdit" name="password_edit">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>170</y>
     <width>211</width>
     <height>20</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>password</string>
   </property>
  </widget>
  <widget class="PushButton" name="login_button">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>230</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>登录</string>
   </property>
  </widget>
  <widget class="BodyLabel" name="password_label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>150</y>
     <width>50</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>密码</string>
   </property>
  </widget>
  <widget class="BodyLabel" name="account_label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>90</y>
     <width>50</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>账号</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>BodyLabel</class>
   <extends>QLabel</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>PushButton</class>
   <extends>QPushButton</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>LineEdit</class>
   <extends>QLineEdit</extends>
   <header>qfluentwidgets</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
